docs/developer-docs/security/security-best-practices/dos.mdx

```diff
@@ -4,32 +4,72 @@ keywords: [security, concept, denial of service, dos, cycles, captcha]
 
 import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
-# Security best practices: Denial of service
+# Security Best Practices: Denial of Service
 
 <MarkdownChipRow labels={["Intermediate", "Security", "Concept"]} />
 
-## Protect against draining the cycles balance
+## Introduction
 
-### Security concern
+Canisters on the Internet Computer are responsible for managing their own resources, including paying for execution, storage, and messaging with cycles. This makes them efficient, but also exposes them to risks such as Denial of Service (DoS) attacks, which can drain their cycles, and the "noisy neighbor" problem, where other canisters consume excessive resources, impacting performance.
 
-Canisters pay for their cycles which makes them inherently vulnerable to attacks that consume all their cycles.
+In this guide, we will explore how to protect your canisters from these risks and manage resources effectively.
 
-### Recommendation
+## 1. Protecting Against DoS and DDoS Attacks
```

Comment:
I'd move this paragraph into the introduction so the rest can become "best practices".